### ISSUE TITLE
Doesn't work on 14.6.1

### ISSUE THREAD

**Issue by sameastham** [assafdori/bypass-mdm#12]
I followed all the directions to the letter, multiple times, and was unable to get this solution to work. Unfortunately now the way the set up screen works is that you have to go through the whole internet process first, then the MDM enrollment process starts, and _then_ you can set up your user ID and password. If you try to stop before that and go back to the recovery screen, it will tell you that there are "no users to recover." I'd be happy to go through the process again and post a sped-up video to demonstrate exactly what I'm talking about if it would be helpful.

**Comment by ccjcan** [comment]
Yes it does I just installed it yesterday. Wipe out everything, reset the Mac, use the revive process and then use this sofware.

**Comment by sameastham** [comment]
Were you on 14.6.1? That is the key here.

**Comment by assafdori** [comment]
Hey @sameastham feel free to attach relevant evidence but it is still working as of now.

**Comment by tanojsaipraveen** [comment]
When I entered password 1234 after this process it was saying wrong password

**Comment by C113293EBC5A43AE99DEA56C40CEE2A6** [comment]
I see this issue too. After first boot, go into recovery, update the hosts file:

```
echo "0.0.0.0 deviceenrollment.apple.com" >>/Volumes/Macintosh\ HD/etc/hosts
echo "0.0.0.0 mdmenrollment.apple.com" >>/Volumes/Macintosh\ HD/etc/hosts
echo "0.0.0.0 iprofiles.apple.com" >>/Volumes/Macintosh\ HD/etc/hosts
```

Then reboot, and go through the setup process until it tells you it cannot fetch the MDM information. Reboot back into recovery, run the script hosted in this repo, and then you should be able to login on next reboot. At least that's what I had to do when I bypass MDM manually. I didn't use this script. I just created an account using dscl.
